gravitee-apim-repository/gravitee-apim-repository-elasticsearch/README.adoc

```diff
@@ -5,8 +5,6 @@
 By default, unit tests are run with a TestContainer version of ElasticSearch 7.17.7, but sometimes it can be useful to run them against other version of ElasticSearch.
```

Comment:
```suggestion
By default, unit tests are run with a TestContainer version of ElasticSearch 7.17.8, but sometimes it can be useful to run them against other version of ElasticSearch.
```